core_planning/twist_filter/src/twist_filter.cpp

```diff
@@ -53,17 +53,34 @@ carma_ros2_utils::CallbackReturn TwistFilter::handle_on_configure(const rclcpp_l
   // parameters on private handles
   get_parameter<double>("config_speed_limit", longitudinal_velocity_limit_);
   longitudinal_velocity_limit_ = longitudinal_velocity_limit_ * 0.44704;
-  
+
   get_parameter<double>("vehicle_acceleration_limit", longitudinal_accel_limit_);
+
+  bool use_sim_time = false;
+  get_parameter<bool>("use_sim_time", use_sim_time);
+
+  auto hardcoded_acceleration_limit = MAX_LONGITUDINAL_ACCEL_HARDCODED_LIMIT_M_S_2;
+
+  // if simulation
+  if (use_sim_time)
+  {
+    hardcoded_acceleration_limit = MAX_SIMULATION_LONGITUDINAL_ACCEL_HARDCODED_LIMIT_M_S_2;
+    RCLCPP_INFO_STREAM(rclcpp::get_logger("logger"), "Simulation mode detected. Accounting hardcoded limit longitudinal acceleration limit used: " << hardcoded_acceleration_limit);
+  }
+  else
+  {
+    RCLCPP_INFO_STREAM(rclcpp::get_logger("logger"), "Accounting hardcoded limit, longitudinal acceleration limit used: " << hardcoded_acceleration_limit);
```

Comment:
Are we able to use `rclcpp::get_logger()` here?

Reply:
replaced with get_logger()